CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 **Note**: Numbers like (\#123) point to closed Pull Requests on the fractal-server repository.
 
+# 1.4.3
+
+* API (internal):
```

Comment:
An API request that was valid in the previous version is now invalid -> this is not an internal change.